tracer.go

```diff
@@ -102,12 +102,13 @@ func NewTracer() ot.Tracer {
 
 // NewTracerWithOptions Get a new Tracer with the specified options.
 func NewTracerWithOptions(options *Options) ot.Tracer {
+	InitSensor(options)
+
 	return NewTracerWithEverything(options, NewRecorder())
 }
 
 // NewTracerWithEverything Get a new Tracer with the works.
 func NewTracerWithEverything(options *Options, recorder bt.SpanRecorder) ot.Tracer {
-	InitSensor(options)
```

Comment:
@pavlobaron Not sure I understand the reason for this move.  Shouldn't `InitSensor` be called from the lowest level so we guarantee that metric collection is initialized regardless of the Tracer instantiation call?